src/reporters/udp_sender.js

```diff
@@ -15,6 +15,7 @@ import dgram from 'dgram';
 import fs from 'fs';
 import path from 'path';
 import {Thrift} from 'thriftrw';
+import {LengthResult} from 'bufrw';
```

Comment:
you are not using this class, no need to import it. You are only using its declaration, which you need to add to `_flow` dir.